os/board/imxrt1020-evk/src/imxrt_bringup.c

```diff
@@ -114,6 +116,38 @@ static void imxrt_i2c_register(int bus)
 }
 #endif
 
+/****************************************************************************
+ * Name: board_pwmm_initialize
+ *
+ * Description:
+ *   PWM intialization for imxrt
+ *
+ ****************************************************************************/
+static void imxrt_pwm_register(int ch)
+{
+#ifdef CONFIG_PWM
+	struct pwm_lowerhalf_s *pwm_ch;
+	char path[16];
+	int ret;
+
+	/* The last two channels(15, 16) multiplex the PINs with LPUART1,
+	 * so for debugging they are not be set to PWM PINs */
+	pwm_ch = imxrt_pwminitialize(ch);
+	if (!pwm_ch) {
+		syslog(LOG_ERR, "Failed to get imxrt PWM Channel lower half\n");
```

Comment:
Let's use dbg or vdbg instead of using syslog directly.
When syslog is disabled, this will be a compilation error.